### ISSUE TITLE
ERROR: Package LIBSVM errored during testing (exit code: 3221226356)

### ISSUE THREAD

**Issue by WuSiren** [JuliaML/LIBSVM.jl#103]
It reports an error when I `test LIBSVM`.

``` julia
ERROR: Package LIBSVM errored during testing (exit code: 3221226356)
```

**Comment by ablaom** [comment]
@WuSiren Thanks for reporting, but could you please provide more context as below. 

I am not able to reproduce the fail in this context:

```
julia> using Pkg; Pkg.status()
Status `/private/var/folders/4n/gvbmlhdc8xj973001s6vdyw00000gq/T/jl_e4ttWI/Project.toml`
  [b1bec4e5] LIBSVM v0.8.0

julia> versioninfo()
Julia Version 1.10.1
Commit 7790d6f0641 (2024-02-13 20:41 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: macOS (x86_64-apple-darwin22.4.0)
  CPU: 12 × Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-15.0.7 (ORCJIT, skylake)
Threads: 12 default, 0 interactive, 6 GC (on 12 virtual cores)
Environment:
  JULIA_LTS_PATH = /Applications/Julia-1.6.app/Contents/Resources/julia/bin/julia
  JULIA_PATH = /Applications/Julia-1.10.app/Contents/Resources/julia/bin/julia
  JULIA_EGLOT_PATH = /Applications/Julia-1.7.app/Contents/Resources/julia/bin/julia
  JULIA_NUM_THREADS = 12
  DYLD_LIBRARY_PATH = /usr/local/homebrew/Cellar/libomp/9.0.1/lib/
  JULIA_NIGHTLY_PATH = /Applications/Julia-1.10.app/Contents/Resources/julia/bin/julia
```

**Comment by ablaom** [comment]
This looks like a known Julia 1.9 issue, fixed in 1.10. See  https://github.com/JuliaML/LIBSVM.jl/issues/95.

@barucden Do you think we should bump julia [compat] to 1.9? I don't know if there a plan for a backwards fix for julia 1.9.

**Comment by barucden** [comment]
The fix (https://github.com/JuliaLang/julia/pull/50135) was backported and released with julia 1.9.2.

I am not against bumping the compatible julia version. An alternative might be to raise a warning for the affected users:
```julia
if Sys.iswindows() and (v"1.8.3" < VERSION < v"1.9.2")
    @warn "LIBSVM is known to error under the current Julia version. Consider upgrading to Julia 1.9.2 (see https://github.com/JuliaML/LIBSVM.jl/issues/95)."
end
```

(Speaking of bumping versions -- we haven't made a release (8.0.1) with the LIBLINEAR version bumped to 0.7. Should we release that?)

**Comment by ablaom** [comment]
> Speaking of bumping versions -- we haven't made a release (8.0.1) with the LIBLINEAR version bumped to 0.7. Should we release that?

That would be nice. 

> The fix (https://github.com/JuliaLang/julia/pull/50135) was backported and released with julia 1.9.2.

Ah, thanks for that. I see @WuSiren was indeed using 1.9.0. In that case I don't think we need to take any further action and can close this. I think it's fair to assume users are using the latest patch of any package.

**Comment by barucden** [comment]
Alright. Closing as duplicate of #95.
